src/lib/Rule/ReplaceInterfaceRector.php

```diff
@@ -55,7 +55,7 @@ public function getNodeTypes(): array
     /**
      * @param \PhpParser\Node\Stmt\Class_ $node
      */
-    public function refactor(Node $node): ?Node
+    public function refactor(Node $node): Node
```

Comment:
Note: this is unrelated to the bug, it's something that came up from PHPStan - method never returns null so, due to covariance, can be narrowed down.